core/src/Cuda/Kokkos_CudaSpace.cpp

```diff
@@ -339,7 +346,14 @@ void CudaSpace::impl_deallocate(
   }
 
   try {
+#ifndef CUDART_VERSION
+#no CUDART_VERSION
```

Comment:
Same comment as above.  `#no` is not a preprocessor directive as far as I know.

Reply:
Thanks for the note, changed here: https://github.com/kokkos/kokkos/pull/4026/commits/3d80e7166e40d30d75d358a9565d18f6bb8dbe4f